### PR TITLE
seperated analysis class, applied pep style guidlines to most of the …

### PR DESCRIPTION
**Pull request by dieggo111**
…code, inserted comments and suggestions... not merged with ur recent branch, because of all the changes. Look for #COMMENTS: ... in all files and check out if you like it.

### CODE REVIEW

File: utilities.py [hunk 1]
```diff
@@ -118,26 +120,30 @@ def save_all_plots(name, folder, figs=None, dpi=200):
 
 class NoStdStreams(object):
     """Surpresses all output of a function when called with with """
-    def __init__(self,stdout = None, stderr = None):
```
Comment (Chilldose) [inline utilities.py:121]: Python developer raten dazu zwischen = immer spaces zu machen um die Leserlichkeit zu erhöhen.

**Review by Chilldose — APPROVED** [review]
Most things you did are cool.

Just a few remarks on coding style, from my point of view.
Every file with less then 2000 lines of code are in my point of view baby files. But if you say you can read the code better in different files, I am fine with that. 

Your changes of the print statements are interesting since I have canged them in the master branch already. Did you have an outdated version?

Thanks for your remarks and work!

File: utilities.py (hunk 1 above)
Comment (dieggo111) [reply utilities.py:121]: pep style ist hier ganz klar: zwischen kwargs keine leerstelle, aber bei allen anderen fällen immer. Also zB a = 123 ...

Comment (Chilldose) [reply utilities.py:121]: ah ok, das wusst ich nicht

**Pull request closed without merging**